sdk/keyvault/azure-security-keyvault-administration/swagger/src/main/java/RbacCustomizations.java

```diff
@@ -0,0 +1,69 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+import com.azure.autorest.customization.Customization;
+import com.azure.autorest.customization.LibraryCustomization;
+import com.github.javaparser.StaticJavaParser;
+import com.github.javaparser.ast.Modifier;
+import com.github.javaparser.ast.body.ClassOrInterfaceDeclaration;
+import org.slf4j.Logger;
+
+import java.net.MalformedURLException;
+import java.net.URL;
+
+/**
+ * Contains customizations for Azure KeyVault's RBAC swagger code generation.
+ */
+public class RbacCustomizations extends Customization {
+    @Override
+    public void customize(LibraryCustomization libraryCustomization, Logger logger) {
+        libraryCustomization.getPackage("com.azure.security.keyvault.administration.models")
+            .getClass("KeyVaultRoleScope")
```

Comment:
Is there a particular reason that only the `KeyVaulRoleScope` class needed this customization?

Reply:
Yeah, it had custom `fromUrl` methods added to the `ExpandableStringEnum` type which aren't common.